src/libraries/System.Net.Ping/src/System/Net/NetworkInformation/Ping.PingUtility.cs

```diff
@@ -24,6 +24,12 @@ private Process GetPingProcess(IPAddress address, byte[] buffer, int timeout, Pi
                 throw new PlatformNotSupportedException(SR.net_ping_utility_not_found);
             }
 
+            // ping utility does not support specifying custom payload
```

Comment:
That is not necessarily true:
`ping 10.211.55.2 -p 0A0B0C0D -s 4`

But it is trickier as there are limits, the option may not exist (busybox?) or may by different. 
```
       -p pattern
           You may specify up to 16 “pad” bytes to fill out the packet you send. This is useful for diagnosing data-dependent problems in a network. For example, -p ff will cause the sent packet to be filled with all ones.
```

Reply:
I did not want to be so specific :)

Reply:
I think it is ok for new. We can write issue for tracking if we want to make improvements. 
You can update the comment setting to something like "We don't support buffers with ping utility"